third_party/rust/Cargo.toml

```diff
@@ -51,54 +53,90 @@ serde_json = "1"
 sha2 = "0.9"
 small_ctor = "0.1"
 static_assertions = "1"
-strobe-rs = "0.8"
+strobe-rs = "0.8.1"
 tempfile = "3"
-thiserror = "1"
-url = "2"
-urlencoding = "1"
+thiserror = "^1.0.30"
+url = "2.2"
+urlencoding = "1.1.1"
 wycheproof = "0.4"
-zeroize = "1"
 
 [dependencies.adblock]
 default-features = false
-version = "0.7"
+version = "0.7.9"
 features = ["full-regex-handling", "debug-info", "css-validation"]
 
 [dependencies.bigdecimal]
 version = "0.1"
 features = ["serde"]
 
+[dependencies.bitvec]
+default-features = false
```

Comment:
Looks like we don't use the alloc-dependent `BitVec` type, and the compiler should tell us if we need atomic for thread-safety. Ok.

Reply:
Not sure what you mean here, this was produced by `gnrt gen` and is required by `ff` and `sta_rs`

Reply:
there does seem to be a mismatch between what's in BUILD.gn and what's in Cargo.toml. The BUILD.gn is what matters, but I'll update Cargo.toml to match